src/add_new.js

```diff
@@ -1,24 +1,34 @@
-const addNew = (leaderboard, name, score, num) => {
-  const leaderboardItem = document.createElement('li');
-  leaderboardItem.classList.add('flex-box', 'item');
-  if (num % 2 === 0) {
-    leaderboardItem.classList.add('even');
-  } else {
-    leaderboardItem.classList.add('odd');
-  }
+const addNew = async (name, points) => {
+  // const leaderboardItem = document.createElement('li');
+  // leaderboardItem.classList.add('flex-box', 'item');
+  // if (num % 2 === 0) {
+  //   leaderboardItem.classList.add('even');
+  // } else {
+  //   leaderboardItem.classList.add('odd');
+  // }
 
-  const nameContainer = document.createElement('span');
-  nameContainer.classList.add('item-player');
-  nameContainer.innerHTML = name;
+  // const nameContainer = document.createElement('span');
+  // nameContainer.classList.add('item-player');
+  // nameContainer.innerHTML = name;
 
-  const scoreContainer = document.createElement('span');
-  scoreContainer.classList.add('item-score');
-  scoreContainer.innerHTML = score;
+  // const scoreContainer = document.createElement('span');
+  // scoreContainer.classList.add('item-score');
+  // scoreContainer.innerHTML = score;
 
-  leaderboardItem.appendChild(nameContainer);
-  leaderboardItem.appendChild(scoreContainer);
+  // leaderboardItem.appendChild(nameContainer);
+  // leaderboardItem.appendChild(scoreContainer);
 
-  leaderboard.appendChild(leaderboardItem);
+  // leaderboard.appendChild(leaderboardItem);
```

Comment:
#### Optional suggestion

- [ ] Kindly, remove the commented lines in your code to look more clean and professional

Reply:
Thank you! I totally forgot about the commented lines. I will make sure to remove them before merging.

Reply:
Changes completed!